__tests__/override-angular-versions.test.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  packageJsonWithV8Dependencies,
+  packageJsonWithV8DevDependencies,
+  packageJsonWithV8All,
+  fullPackageJson,
+  packageUnmatching
+} from './package-json-versions';
+import {overrideAngularVersions} from '../src/override-angular-versions';
+import {getAngularVersions} from '../src/get-angular-versions';
+
+describe(overrideAngularVersions.name, () => {
```

Comment:
Add a test case that verifies that the `ng-packagr` builder is not added when it didn't exist in the project configuration.